lib/maze/hooks/appium_hooks.rb

```diff
@@ -76,7 +71,7 @@ def after_all
           Maze::BrowserStackUtils.stop_local_tunnel
         elsif Maze.config.farm == :bb
           Maze::BitBarUtils.stop_local_tunnel
-          Maze::BitBarUtils.release_account(Maze.config.tms_uri)
+          Maze::BitBarUtils.release_account(Maze.config.tms_uri) if ENV['BUILDKITE']
```

Comment:
We should probably think about n approach for doing this that doesn't rely on checking the environment variable in (potentially) various part of the code.  It's an existing problem (that I created!) for logging, though, so no action needed here.  Just something to think about.

Reply:
How's about a configuration option which defaults to False but is overwritten by the `BUILDKITE` env var?  I think it should be in a separate PR however